client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const _ = require("lodash");
```

Comment:
IE11 - i'm pretty sure if you don't support it, you don't have to transform ES6 to ES5